Testing/functional/tests/cypress/integration/ui/user/dependents.js

```diff
@@ -16,7 +16,22 @@ describe("COVID-19", () => {
         cy.intercept("GET", "**/UserProfile/*/Dependent", {
             fixture: "UserProfileService/dependent.json",
         });
-        cy.enableModules(["CovidLabResults", "Laboratory", "Dependent"]);
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "laboratory",
+                    enabled: true,
+                },
```

Comment:
This dataset can be removed.  It's now "covid19TestResult" which is already enabled here.